CSC8503/InventoryBuffSystem/PlayerInventory.cpp

```diff
@@ -0,0 +1,93 @@
+	#include "PlayerInventory.h"
+
+	using namespace InventoryBuffSystem;
+
+	void PlayerInventory::Init()
+	{
+		for (int playerNo = 0; playerNo < 4; playerNo++)
```

Comment:
Probably should remove the "magic number" of 4, make it a constexpr of PLAYER_MAX